NetworkConfigurations/TrafficControlAndPolicingSubCategories/TrafficShaping.md

```diff
@@ -0,0 +1,417 @@
+---
+title: Traffic Shaping
+grand_parent: Network Configuration
+parent: Traffic Control and Policing
+nav_order: 3
+layout: default
+---
+
+# Traffic Shaping
```

Comment:
How did we verify the configurations below are functional and perform as intended?

Reply:
Verified using `iperf` utility and listed the outputs at https://github.com/dentproject/docs.dent.dev/pull/41/commits/cff727ccb266a2d909d61fbbbdbbdeb27d98b02e.